### ISSUE TITLE
(???) Combinations I want to make sometimes being interrupt by the game

### ISSUE THREAD

**Issue by mesosoi** [Silverware-Games/matchy-star-issues#5]
When I move my cursor from one element to the another one, the game should detect when I stop holding the left key. Because as for now I can't create combinations I want, the game just selects the combination which is nearest to my cursor.

**Comment by michaelplzno** [comment]
Please explain this a bit more, I don't understand.

**Comment by mesosoi** [comment]
> Please explain this a bit more, I don't understand.

When I am moving an icon on the field to match 3-in-a-row, i can just move my cursor anywhere in the field and the game will automatically find the best combination. The game should detect when I am stopping holding my mouse left key and only after that actually making a combination.

**Comment by michaelplzno** [comment]
I'm not aware of any "best match" code I wrote but maybe I forgot. It drops the gem at the closest square to where it currently is and detects a swap to a new spot when you move it out of the square in question. I'll play with this and make a video.

**Comment by mesosoi** [comment]
> I'm not aware of any "best match" code I wrote but maybe I forgot. It drops the gem at the closest square to where it currently is and detects a swap to a new spot when you move it out of the square in question. I'll play with this and make a video.

Yes, this is what I mean. And this feature sucks. Because sometimes I want to create own combinations which aren't always in the nearest square to the cursor.

**Comment by michaelplzno** [comment]
let me make a video so we are on the same page.
